features/base/participants/actions.js

```diff
@@ -9,6 +11,46 @@ import {
 } from './actionTypes';
 import './reducer';
 
+
+/**
+ * Returns the URL of the image for the avatar of a particular user,
+ * identified by its id.
+ * @param {string} userId - user id
+ * @param {string} email - user email
+ */
+function getAvatarUrl(userId, email) {
+    // TODO: Use disableThirdPartyRequests config
+
+    let avatarId = email || userId;
+
+    // If the ID looks like an email, we'll use gravatar.
+    // Otherwise, it's a random avatar, and we'll use the configured
+    // URL.
+    let random = !avatarId || avatarId.indexOf('@') < 0;
+
+    if (!avatarId) {
```

Comment:
!avatarId is checked for the second time here. I'd rather have an if/else that does a single check,